packages/trading-widget/src/trading-widget/components/widget/widget-input/asset-composition-table/asset-composition-table.tsx

```diff
@@ -21,36 +26,45 @@ export const AssetCompositionTable = ({
         >
           <tbody>
             {poolComposition.map(
-              ({ fraction, fractionUsd, tokenName, asset }) => (
-                <tr key={tokenName}>
-                  <td
-                    style={{
-                      width: 150,
-                      maxWidth: 150,
-                    }}
-                  >
-                    <TokenBadge
-                      symbol={tokenName}
-                      iconSymbols={asset.iconSymbols}
-                      symbolClasses="dtw-text-xs dtw-font-bold"
-                      size={iconSize}
-                    />
-                  </td>
-                  {showFraction && (
+              ({ fraction, fractionUsd, tokenName, asset, tokenAddress }) => {
+                const isLeveragedRethAsset =
+                  isFlatMoneyLeveragedRethAsset(tokenAddress)
+                return (
+                  <tr key={tokenName}>
                     <td
                       style={{
                         width: 150,
                         maxWidth: 150,
                       }}
                     >
-                      <p className="dtw-truncate">{fraction}</p>
-                      <p className="dtw-truncate dtw-text-xs dtw-text-[color:var(--panel-secondary-content-color)]">
-                        {fractionUsd}
-                      </p>
+                      <TokenBadge
+                        symbol={tokenName}
+                        iconSymbols={asset.iconSymbols}
+                        symbolClasses="dtw-text-xs dtw-font-bold"
+                        size={iconSize}
+                      />
+                      {isLeveragedRethAsset && (
+                        <div className="dtw-flex dtw-gap-1 dtw-items-center dtw-text-xs dtw-text-[color:var(--panel-secondary-content-color)]">
+                          As <TokenIcon symbols={['reth']} size="xs" /> rETH
```

Comment:
Can `tokenName` work here instead of symbol hardcode?
And please move `As` to translation key.

Reply:
Updated
Moved rETH to `FLATMONEY_COLLATERAL_SYMBOL_MAP`